### PR TITLE
Docker: execute docker_adduser.sh if USER is not root

### PR DESCRIPTION
**Pull request by SeasoulChris**
execute docker_adduser.sh if USER is not root

### CODE REVIEW

File: docker/scripts/dev_start.sh [hunk 1]
```diff
@@ -424,9 +424,7 @@ function main(){
     fi
     set +x
 
-    # User with uid=1000 or username=apollo excluded
-    if [[ "${USER}" != "root" ]] && [[ "${USER}" != "apollo" ]] \
-        && [[ $USER_ID -ne 1000 ]]; then
+    if [[ "${USER}" != "root" ]]; then
```
Comment (xiaoxq) [inline docker/scripts/dev_start.sh:427]: We may need to keep the `"${USER}" != "apollo"` part, as there is already a `apollo` user in the image, @storypku correct?

Comment (SeasoulChris) [reply docker/scripts/dev_start.sh:427]: here is a situation: if  default user_name of system is `apollo`, when he execute dev_start.sh, docker_adduser.sh will not be executed，then some devices such as ttyACM* will not get proper permissions

Comment (SeasoulChris) [reply docker/scripts/dev_start.sh:427]: sadly , now the default system user_name of D-kit is apollo

Comment (storypku) [reply docker/scripts/dev_start.sh:427]: > We may need to keep the `"${USER}" != "apollo"` part, as there is already a `apollo` user in the image, @storypku correct?

Yeap. So we should re-factore `docker_add_user.sh` asap to fix https://github.com/ApolloAuto/apollo/issues/11296

**Pull request closed without merging**